src/Microsoft.TemplateEngine.Cli/Dotnet.cs

```diff
@@ -76,6 +87,12 @@ public static Dotnet AddProjectsToSolution(string solutionFile, IReadOnlyList<st
                 "add"
             };
 
+            if (!string.IsNullOrEmpty(solutionFolder))
```

Comment:
I've suggested previously the best practice of instead using string.IsNullOrWhiteSpace